src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -33,6 +33,7 @@ Give your alert a descriptive name, such as the team affected and the topic of t
 - Changes state from `ignored` to `unresolved`
 - Seen more than a certain number of times in an interval
 - Seen by more than a certain number of unique users in an interval
+- *Beta* An issue affects more than {X} percent of [sessions](/product/releases/health/#session) in {time}. This is an approximation and will trigger when the ratio of the issue’s frequency to the number of sessions exceeds the threshold.
```

Comment:
Does "beta" mean EA or something else? If it means EA, I've added an EA note in my suggested edit. If it means something else, then we might need to format it differently because it doesn't stand out enough in italics. Also wondering if you need everything after "approximation" as the initial definition seems pretty clear. 

```suggestion
- An issue affects more than {X} percent of [sessions](/product/releases/health/#session) in {time}. This is an approximation and will trigger when the ratio of the issue’s frequency to the number of sessions exceeds the threshold.

<Note>

The "percent of sessions" alert trigger is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>
```